src/accounts/AccountPage.tsx

```diff
@@ -1,49 +1,45 @@
 // Libraries
 import React, {ChangeEvent, FC, useContext, useEffect, useState} from 'react'
 import {useSelector} from 'react-redux'
+import {Switch, Route} from 'react-router-dom'
 
 import {
   Button,
-  ButtonShape,
-  ComponentColor,
+  // IconFont,
   ComponentSize,
-  ConfirmationButton,
   FlexBox,
   FlexDirection,
   Input,
   InputType,
   Overlay,
   Page,
 } from '@influxdata/clockface'
+import {DeleteFreeAccountButton} from 'src/accounts/DeleteFreeAccount'
+import DeleteAccountOverlay from 'src/organizations/components/DeleteAccountOverlay'
 
-import {getMe} from 'src/me/selectors'
-import {UsersContext, UsersProvider} from 'src/users/context/users'
+// Selectors
+import {selectCurrentIdentity} from 'src/identity/selectors'
+
+// Constants
+import {CLOUD} from 'src/shared/constants'
 
 // Utils
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
 import {UserAccountContext} from 'src/accounts/context/userAccount'
 import AccountTabContainer from 'src/accounts/AccountTabContainer'
 import AccountHeader from 'src/accounts/AccountHeader'
+import DeleteOrgProvider from 'src/organizations/components/DeleteOrgContext'
```

Comment:
@wdoconnell  now that we're updating terminology, moving from `Delete Org` to `Delete Account`, should we also do the following?

- Update the name of the deletion context from `DeleteOrgContext` to `DeleteAccountContext` 
- Move the file from `../organizations` subfolder to `/accounts` subfolder

Reply:
I think that's right - this belongs in the accounts folder now. I might also be even clearer if this provider is being used only for Deleting free accounts -- call it 'DeleteFreeAccountContext' instead of 'DeleteAccountContext'.  But definitely double check to see if it's used at all in the Contract/PAYG context.